doc/tutorial/extending_theano.txt

```diff
@@ -64,6 +64,13 @@ possibilities you may encounter or need.  For that refer to
         # Properties attribute
         __props__ = ()
 
+        #itypes and otypes attributes are
+        #compulsory if make_node method is not defined.
+        #They're the type of input and output respectively
+        itypes = []
+        otypes = []
```

Comment:
Sorry for the long delay to review this. I got distracted by NIPS and vacations:)

I think this comment is the only thing left. Can you do it?

thanks